src/backend/api/meals.js

```diff
@@ -2,11 +2,96 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+
+// GET api/meals/ query parameters
+
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    if (request.query.maxPrice) {
```

Comment:
Ideally, you should be considering all the different query parameters. Instead you are currently only considering one query parameter at a time. For instance, it should be possible to say `?maxPrice=100&title=pizza` to only get back meals where the max price is 100 AND where the title is pizza.

Reply:
Doesn't seem like you worked on this feedback.